src/main/java/org/apache/datasketches/SetOperationCornerCases.java

```diff
@@ -0,0 +1,188 @@
+    ResultDegen_New(05,         "A{>1.0, 0, F} ; B{ 1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.RESULTDEGEN_THA_0_F),   //5
+    ResultDegen_Exact(06,       "A{>1.0, 0, F} ; B{ 1.0,>0, F}",
+        IntersectResult.RESULTDEGEN_MIN_0_F, AnotbResult.RESULTDEGEN_THA_0_F), //6
+
+    NewDegen_ResultDegen(010,   "A{>1.0, 0, T} ; B{>1.0, 0, F}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),     //8
+    NewDegen_NewDegen(011,      "A{>1.0, 0, T} ; B{>1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),     //9
+    NewDegen_Estimation(012,    "A{>1.0, 0, T} ; B{>1.0,>0, F}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //10
+    NewDegen_New(015,           "A{>1.0, 0, T} ; B{ 1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //13
+    NewDegen_Exact(016,         "A{>1.0, 0, T} ; B{ 1.0,>0, F}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //14
+
+    Estimation_ResultDegen(020, "A{>1.0,>0, F} ; B{>1.0, 0, F}",
+        IntersectResult.RESULTDEGEN_MIN_0_F, AnotbResult.SKA_TRIM),   //16
+    Estimation_NewDegen(021,    "A{>1.0,>0, F} ; B{>1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.SKETCH_A),     //17
+    Estimation_Estimation(022,  "A{>1.0,>0, F} ; B{>1.0,>0, F}",
+        IntersectResult.FULL_INTER, AnotbResult.FULL_ANOTB),  //18
+    Estimation_New(025,         "A{>1.0,>0, F} ; B{ 1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.SKETCH_A),     //21
+    Estimation_Exact(026,       "A{>1.0,>0, F} ; B{ 1.0,>0, F}",
+        IntersectResult.FULL_INTER, AnotbResult.FULL_ANOTB),  //22
+
+    New_ResultDegen(050,        "A{ 1.0, 0, T} ; B{>1.0, 0, F}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //40
+    New_NewDegen(051,           "A{ 1.0, 0, T} ; B{>1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //41
+    New_Estimation(052,         "A{ 1.0, 0, T} ; B{>1.0,>0, F}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //42
+    New_New(055,                "A{ 1.0, 0, T} ; B{ 1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //45
+    New_Exact(056,              "A{ 1.0, 0, T} ; B{ 1.0,>0, F}",
+        IntersectResult.NEW_1_0_T, AnotbResult.NEW_1_0_T),    //46
+
+    Exact_ResultDegen(060,      "A{ 1.0,>0, F} ; B{>1.0, 0, F}",
+        IntersectResult.RESULTDEGEN_MIN_0_F, AnotbResult.SKA_TRIM),   //48
+    Exact_NewDegen(061,         "A{ 1.0,>0, F} ; B{>1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.SKETCH_A),     //49
+    Exact_Estimation(062,       "A{ 1.0,>0, F} ; B{>1.0,>0, F}",
+        IntersectResult.FULL_INTER, AnotbResult.FULL_ANOTB),  //50
+    Exact_New(065,              "A{ 1.0,>0, F} ; B{ 1.0, 0, T}",
+        IntersectResult.NEW_1_0_T, AnotbResult.SKETCH_A),     //53
+    Exact_Exact(066,            "A{ 1.0,>0, F} ; B{ 1.0,>0, F}",
+        IntersectResult.FULL_INTER, AnotbResult.FULL_ANOTB);  //54
```

Comment:
Some of the descriptions for degenerative sketches use >1.0 for Theta which looks like a typo.

Reply:
Thank you!